src/json/data_techexposure_company_companies.json

```diff
@@ -1,23 +1,27 @@
 [
 	{
-		"id": "Nissan Motor Co., Ltd.",
+		"id": "Hyundai Motor Co., Ltd.",
 		"ald_sector": "Automotive",
 		"technology": "Electric",
-		"plan_tech_share": 0.207746608894222,
-		"port_weight": 0.00761768898721195,
-		"scenario": "NZE 2050",
+		"plan_tech_share": 0.1927,
+		"port_weight": 0.1156,
+		"allocation": "portfolio_weight",
+		"scenario_source": "WEO2023",
+		"scenario": "APS",
 		"year": 2028,
 		"asset_class": "Listed Equity",
 		"ald_sector_translation": "Automotive",
 		"asset_class_translation": "Listed Equity",
 		"technology_translation": "LDV: Electric"
 	},
```

Comment:
Hyundai appears to be accurately classed as an `Automotive` company in the data here?